docs/jwst/pathloss/reference_files.rst

```diff
@@ -1,137 +1,192 @@
 Keyword Value      Comment
-======= ========== =========================================================================================
+======= ========== ==============================================================================
 CRPIX1  1.0        Reference pixel in fastest dimension
 CRVAL1  -0.5       Coordinate value at this reference pixel
 CDELT1  0.05       Change in coordinate value for unit change in index
 CTYPE1  'UNITLESS' Type of physical coordinate in this dimension
-======= ========== =========================================================================================
+======= ========== ==============================================================================
 
-This dimension expresses the decenter along the dispersion direction for a point source
+Dimension 2 expresses the decenter along the direction perpendicular to the dispersion
+for a point source:
 
-======= ========== =========================================================================================
+======= ========== ==============================================================================
 CRPIX2  1.0        Reference pixel in fastest dimension
 CRVAL2  -0.5       Coordinate value at this reference pixel
 CDELT2  0.05       Change in coordinate value for unit change in index
 CTYPE2  'UNITLESS' Type of physical coordinate in this dimension
-======= ========== =========================================================================================
+======= ========== ==============================================================================
 
-This dimension expresses the decenter along the direction perpendicular to the dispersion for a point source
+Dimension 3 expresses the change of correction as a function of wavelength:
 
-======= ========== =========================================================================================
+======= ========== ==============================================================================
 CRPIX3  1.0        Reference pixel in fastest dimension
 CRVAL3  6.0E-7     Coordinate value at this reference pixel
 CDELT3  2.35E-7    Change in coordinate value for unit change in index
-CTYPE3  'Meter'    Type of physical coordinate in this dimension (should be 'WAVELENGTH')
-======= ========== =========================================================================================
-
-This dimension expresses the change of correction with wavelength
+CTYPE3  'METER'    Type of physical coordinate in this dimension
+======= ========== ==============================================================================
 
 The NIRISS SOSS reference file should also have WCS components, but their
-interpretation is different from those in the NIRSPEC reference file:
+interpretation is different from those in the NIRSpec reference file.
+Dimension 1 expresses the decenter along the dispersion direction for a point source:
 
```

Comment:
This should be:
Dimension 1 expresses the column number in the science data.

Reply:
Fixed in latest commit